README.md

```diff
@@ -12,6 +12,9 @@ It lets you easily add real-time live chat and in-app messaging in your websites
 
 Kommunicate has a powerful human + chatbot hybrid platform, where you can integrate Dialogflow and Amazon Lex chatbot without using any code. You can also integrate any third-party or custom chatbot in Kommunciate as well. The automatic bot to human handoff enabled, customizable and rich-messaging equipped chat-widget gives you ample options to make your chatbot more powerful. 
 
+![Chatwidget](docs/assets/web-chatwidget.png)
```

Comment:
@PoojaGC You need to have a proper URL to load this image.
There is no "doc" folder in this repository, so no image will load here.

Reply:
May I know where can I upload the file to show it here?